core/unit_test/CMakeLists.txt

```diff
@@ -455,27 +455,12 @@ if(Kokkos_ENABLE_SYCL)
        ${CMAKE_CURRENT_BINARY_DIR}/sycl/TestSYCL_Atomics.cpp
        ${CMAKE_CURRENT_BINARY_DIR}/sycl/TestSYCL_Atomics.cpp
        ${CMAKE_CURRENT_BINARY_DIR}/sycl/TestSYCL_AtomicViews.cpp
-       ${CMAKE_CURRENT_BINARY_DIR}/sycl/TestSYCL_BlockSizeDeduction.cpp
```

Comment:
But then I at least know when it is worth looking at it again. The name of the test doesn't give that away.

Reply:
I accept the point you make.  In general I would object that it can give a false sense of near feature completeness when looking at the list of unit tests that are removed.  But I can see how this is somehow "tagging" the unit test and on top of that this test is kinda silly anyway.